### DISCUSSION TITLE
Tabs fullwidth en desktop

### DISCUSSION THREAD

**Discussion by aweell** [Telefonica/mistica-design#656] [Proposals]
Ahora mismo las tabs crecen según su contenido (texto), ¿podría ser opcional que esas tabs ocuparan todo el hueco disponible hasta ocupar el tamaño entero del viewport? sería el mismo comportamiento que en mobile, este es el diseño que tenemos:
 
https://www.figma.com/file/9NyPqmtgsCZnLisRNyn4ZF/Flujos-market-place?node-id=1986%3A46148
 
ahora mismo con el componente Tabs tal cual esta quedaría así:
https://mistica-web.vercel.app/playroom#?code=N4Igxg9gJgpiBcIA8AhCAPABABwIZSgEsA7AcwF5gBGANgF8A%2BAHWM0yQBVcAjAZxbZteMADYwwAFxhQAksVjpKABjoDBwsZOlyFlUjAkBlCbikAKJiBPcdMdJYA0mJQEpVrQRGIBhABa4yGEphIxNzS2tbexA3NTZrXkoAbWBMKXQJeExLbwhNMEIvGF5MM1dLTDonVPTM7JAAOQAxDhKANRh5QigIErKXCroAXXc2AHpmYiQxtHRJllQMHHwiMkpaRjUkORESGExePDAg4ABmACZNjzYkAEEAN1NcACcDwgAvE6pzgA46A%2BeYHIll8Egk2F48DGY0IADpCABbUgAV2esMgCLGxAASigAKIAWQJbVh2DIFW4%2BH0lFO-wmcXYHDsElOmFwiUs2EsDAAqsJnsRcAiYNMmRlTpMbmMdntJtNZpMQA4QAB3boSXy8BBJKinGg-EZAA

**Comment by aweell** [comment]
Actualmente está planteada una [evolución del componente](https://github.com/Telefonica/mistica-design/issues/627) con el objetivo de resolver algunos problemas de accesibilidad.

**Comment by yceballost** [comment]
Tendría sentido que el componente Tab tuviese una prop `fullwidth` para poder conseguir lo que Metaverso necesita? Cuál es exactamente la necesidad que hay detrás de ese fullwidth? por qué no le funciona lo que hay actualmente? (con los arreglos que necesita, claro)

**Comment by yceballost** [comment]
@eziomenutti @poloba no hemos encontrado razones para que el componente `Tab` tenga que tener el comportamiento de fullwidth, creemos que el comportamiento actual funciona perfectamente y no aporta mucho el fullwidth.

Por otro lado, también parece que acerca el visual más a un entorno de tablet que a uno de desktop cuando las usas a fullwidth.

Si tenéis algún dato o información que nos ayude a entender bien por qué el uso actual no cubre la necesidad, añádidlo aquí porfa!

Eso sí, tenemos que solucionar que la tab se vaya a dos líneas, eso es un bug del componente y tenemos que arreglarlo.
